src/test/java/edu/kis/vh/nursery/RyhmersJUnitTest.java

```diff
@@ -36,9 +36,7 @@ public void testIsFull() {
 		//which doesn't have any fixed capacity, we should test IntArrayStack instead.
 		DefaultCountingOutRyhmer ryhmer = new DefaultCountingOutRyhmer(new IntArrayStack());
 		final int STACK_CAPACITY = 12;
-		//We need to fix this loop condition because we don't want to fill the stack completely
-		//before asserting if it's empty.
-		for (int i = 0; i < STACK_CAPACITY - 1; i++) {
+		for (int i = 0; i < STACK_CAPACITY; i++) {
```

Comment:
Is it a correct approach to fixing errors showed by tests?
Errors are still within the project. STACK_CAPACITY is 12 so 12 elements should be added to the stack.